components/alert-button/alert.css

```diff
@@ -0,0 +1,68 @@
+alert-container{
+    display:flex;
+    flex-flow:wrap column;
+    align-items:flex-start;
+    margin-left:0;
+    border:1px solid grey;
+    padding:25px;
+    width:fit-content;
+    background-color: rgb(248, 244, 244);
+      
+}
+
+.alert{
+    width:400px;
+    padding: 10px;
+    text-align:left;
```

Comment:
can use flex start or flex end property here

Reply:
i have the changes ,thankyou for the review .Please Approve PR